packages/web3-common/src/formatter.ts

```diff
@@ -60,9 +60,9 @@ export type DataFormat = {
 	readonly bytes: FMT_BYTES;
 };
 
-export const DEFAULT_RETURN_FORMAT = { number: FMT_NUMBER.HEX, bytes: FMT_BYTES.HEX } as const;
+export const DEFAULT_RETURN_FORMAT = { number: FMT_NUMBER.BIGINT, bytes: FMT_BYTES.HEX } as const;
+export const ETH_DATA_FORMAT = { number: FMT_NUMBER.HEX, bytes: FMT_BYTES.HEX } as const;
```

Comment:
```suggestion
export const ETH_INPUT_DATA_FORMAT = { number: FMT_NUMBER.HEX, bytes: FMT_BYTES.HEX } as const;
```
I think should rename its all occurrences for better readability.

Reply:
@jdevcs As per my knowledge, Eth implementation follows the same format for input/output. In that aspect I named this value as `ETH_DATA_FORMAT`. Please suggest if you think `ETH_INPUT_DATA_FORMAT` is better usage?